integrationExamples/gpt/userId_example.html

```diff
@@ -184,9 +184,16 @@
                         name: 'idl_env',
                         expires: 30
                       }
+                    }, {
+                      name: "sharedId",
+                      storage: {
+                        type: "cookie",
+                        name: "sharedId",
```

Comment:
```suggestion
                        name: "sharedid",
```
it looks like storage.name is lowercase in all other examples, we should probably keep that consistency.